automatic/slack/tools/chocolateyInstall.ps1

```diff
@@ -23,5 +23,16 @@ $packageArgs = @{
   validExitCodes= @(0)
 }
 
-Install-ChocolateyPackage @packageArgs
+$SlackPath = Join-Path -Path $Env:ProgramFiles -ChildPath 'Slack\slack.exe'
+$SlackPresent = Test-Path -Path $SlackPath
 
+if ($SlackPresent) {
+  $InstalledVersion = (Get-ItemProperty -Path $SlackPath -ErrorAction:SilentlyContinue).VersionInfo.ProductVersion
+  $SlackOutdated = [Version]$($Env:ChocolateyPackageVersion) -gt [Version]$InstalledVersion
+}
+
+# Only Attempt an install if the existing version is the same or newer than the package version
+if (-not $SlackPresent -or ($SlackPresent -and $SlackOutdated))
```

Comment:
This will break a forced install of the same Slack version.

Reply:
Fair, though I would just uninstall and re-install; this does not follow the expected behavior of `--force`, I have submitted the revision.